src/promptflow-tools/tests/test_common.py

```diff
@@ -77,3 +77,16 @@ def test_parse_function_role_prompt(self):
     def test_success_parse_role_prompt(self, chat_str, expected_result):
         actual_result = parse_chat(chat_str)
         assert actual_result == expected_result
+
+    @pytest.mark.parametrize(
+        "chat_str, expected_result",
+        [
+            ("\nsystem:\nname:\nAI \n content:\nfirst\n\nuser:\nsecond", [
+                {'role': 'system', 'name': 'AI', 'content': 'first'}, {'role': 'user', 'content': 'second'}]),
+            ("\nsystem:\nname:\n\n content:\nfirst\n\nuser:\nname:\n\nperson\n content:\n", [
+                {'role': 'system', 'name': '', 'content': 'first'}, {'role': 'user', 'name': 'person', 'content': ''}])
+        ]
+    )
+    def test_success_parse_name_in_role_prompt(self, chat_str, expected_result):
```

Comment:
more than success tests. let's also test fallback logic.

Reply:
add more tests and rename test